### ISSUE TITLE
The new org.chromium.bootstrap 1.1.2 breaks our builds

### ISSUE THREAD

**Issue by espy** [MobileChromeApps/mobile-chrome-apps#512]
Hello!

We're a bit stumped, yesterday's org.chromium.bootstrap 1.1.2 release breaks our builds 
````
FAILURE: Build failed with an exception.

* Where:
Script '/Users/espy/Sites/ehealth/lr-sense-ebola-followup/build/sense-ebola/platforms/android/org.chromium.bootstrap/mgsenseebola-cca.gradle' line: 5

* What went wrong:
A problem occurred evaluating script.
> Could not find property 'cdvDebugSigningPropertiesFile' on root project 'android'.
````
What happens is that our `build.gradle` looks a lot different from the current cordova-android one, especially the bits that deal with the property from the error message [here](https://github.com/apache/cordova-android/blob/98f90340f365a553c089946859926fc07338125d/bin/templates/project/build.gradle#L86).

We've got cca 0.5.1 installed, and yet our `build.gradle` seems to be quite old, it sets the gradle version to 1.12, whereas android-cordova is at 2.2.1. Thing is: we have no clue what `build.gradle` gets generated from, or how we could upgrade gradle (or even definitively determine its version).

So we thought, no big deal, we'll install org.chromium.bootstrap 1.1.1, but it's not in the [plugins registry](http://plugins.cordova.io/#/package/org.chromium.bootstrap), and we couldn't find it anywhere else. 

In short: we either need a way to revert to 1.1.1, or (I assume) a way to upgrade gradle and/or our `build.gradle` and whatever it's generated from, and we can't figure out how to do either. We've reinstalled cca and removed and added the android platform with it, but no dice. 

We'd be very grateful for your help! And thanks for your time :)

**Comment by karlwestin** [comment]
Might be worth mentioning this:
i took the cca module from this repo, init:ed the submodules and then used the current master via `npm link`. Then we were able to build the project alex mentioned above.

So i guess, could you please publish a new version to NPM?

**Comment by mmocny** [comment]
@agrieve was in the process of resolving this today.  Sorry about it!

Basically, he will move our the gradle changes into its own plugin (cca-gradle?), and publish a new cca version which installs that.

For existing cca cli's versions (as you are using) he re-published chrome.bootstrap to remove the gradle changes.

The fact that you couldn't revert manually I assume means that the old version was unpublished from the registry?  Andrew, is this part of our release process right now?

**Comment by karlwestin** [comment]
Thanks!
When the cordova plugin gods in their mercyfulness gives us the new version 1.1.3, we can build. Some machines still pulls in 1.1.2, i guess for some cache reasons, hopefully that will go away in a day or so :)

May i suggest version locking the different plugins in a published NPM version, + change the habit of removing old plugins from the repo? That way, we users can at lease back pedal a little bit on breaking changes like this

**Comment by agrieve** [comment]
Yes! Really sorry about this guys :(.

I released the fixed version shortly after discovering the mess-up, but looks like caching is getting the better of my fix. Please try clearing the cache via:

    rm -r ~/.npm/registry.cordova.io/
    rm -r ~/.plugman/cache

And trying again (it should pick up 1.1.3 now)
